front/src/components/member/MemberUpdateModal.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+  <div class="d-flex flex-column modal-mask" @click.self="closeModal">
+    <VSpacer />
+    <transition name="bounce">
+      <div v-if="rendered" class="pa-6 pb-1 modal-container">
+        <VTextField
+          label="새 닉네임을 입력하세요."
+          color="grey darken-2"
+          v-model="newNickname"
+          @keypress.enter="updateMember"
+        />
+        <div class="button-box">
+          <v-btn
+            text
+            class="mb-2 text-subtitle-1 font-weight-bold grey--text text--darken-1"
+            @click.prevent="closeModal"
+          >
+            취소
+          </v-btn>
+          <v-btn
+            text
+            :disabled="this.nicknameNotChanged()"
```

Comment:
```suggestion
            :disabled="nicknameNotChanged()"
```

vue template 내에서는 this를 붙일 일이 없어요!